src/components/Header/__tests__/Header.test.tsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
 import Header from '../Header';
 
 describe('<Header />', () => {
   it('renders a visible header title and logo', () => {
-    const { getByRole } = render(<Header logo="logo" />);
+    const { getByRole } = render(
+      <MemoryRouter>
+        <Header logo="logo" />
```

Comment:
This prop was updated to load the logo itself instead of being passed a prop
```suggestion
        <Header />
```